hibernate-core/src/main/java/org/hibernate/cfg/CollectionPropertyHolder.java

```diff
@@ -319,7 +319,7 @@ public Join addJoin(JoinTable joinTableAnn, boolean noDelayInPkColumnCreation) {
 
 	@Override
 	public String toString() {
-		return super.toString() + "(" + collection.getRole() + ")";
+		return super.toString() + '(' + collection.getRole() + ')';
```

Comment:
This kind of changes are also fine, and I love you're being thourough with them. But please isolate them into a separate PR and target `main` only.